src/export/word/template.ts

```diff
@@ -39,6 +34,15 @@ export async function defaultTemplate(data: LoadedArticle): Promise<Document> {
     docxState.renderContent(state.doc);
   });
 
+  const references = Object.values(article.references);
+  if (references.length > 0) {
+    docxState.renderContent(createReferenceTitle());
+  }
+  references.forEach(({ state }) => {
+    if (!state) return;
+    docxState.renderContent(state.doc);
+  });
+
```

Comment:
This should probably be a function but I didn't extract it to avoid having side effects on `docxState` be hidden in a function call